Automattic/jetpack
# Command Palette: Add GH deployments is available.

Fixes:
- https://github.com/Automattic/wp-calypso/issues/88345

## Proposed changes:

* Added if GitHub deployments are available.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply these changes to your sandbox.
* Add the `wpcom-github-deployments` sticker to your blog.
* Open the browser's console and type `commandPaletteConfig`.
* You should see that the new property is included and set to true if you have the blog sticker.
